cluster-up/hack/common.sh

```diff
@@ -32,6 +32,7 @@ KUBEVIRT_DEPLOY_GRAFANA=${KUBEVIRT_DEPLOY_GRAFANA:-false}
 KUBEVIRT_CGROUPV2=${KUBEVIRT_CGROUPV2:-false}
 KUBEVIRT_DEPLOY_CDI=${KUBEVIRT_DEPLOY_CDI:-false}
 KUBEVIRT_DEPLOY_CDI_LATEST=${KUBEVIRT_DEPLOY_CDI_LATEST:-false}
```

Comment:
Do we need to keep this variable? Could we not just use? 

`KUBEVIRT_CUSTOM_CDI_VERSION="latest"`

Reply:
Yes, we don't want to always go with "latest", the default is "", i.e. use whatever CDI is in kubevirtci at that point
kubevirt/kubevirt only uses released versions

Reply:
But in the case of wanting to test against latest - could you not just use `KUBEVIRT_CUSTOM_CDI_VERSION="latest"`  instead of needing to set `KUBEVIRT_DEPLOY_CDI_LATEST="true"` 

It would just give us a chance to remove one env variable.

Reply:
oh. I kept that to not break anything that may be already using it..

Reply:
https://github.com/search?q=org%3Akubevirt%20KUBEVIRT_DEPLOY_CDI_LATEST&type=code

We should be good to drop it, as the person that introduced it I really don't mind, it's just as easy to set `KUBEVIRT_CUSTOM_CDI_VERSION="latest"`  as @brianmcarey highlights.

Reply:
Yes -  I did a quick search through project-infra and there no mention of `KUBEVIRT_DEPLOY_CDI_LATEST`

Reply:
makes sense, done